treeherder/perf/alerts.py

```diff
@@ -11,9 +11,15 @@
 
 
 def generate_new_alerts_in_series(signature):
+    # get series data starting from either:
+    # (1) the last alert, if there is one
+    # (2) the alerts max age
+    # (use whichever is newer)
     series = PerformanceDatum.objects.filter(
         signature=signature).order_by(
-            'push_timestamp')
+            'push_timestamp').filter(
```

Comment:
Django does re-order these internally (it does lazy evaluation of querysets), so this is purely aesthetic - but would it be better to have the two `filter()`s adjacent, and the `order_by()` last?